todoapp/test_selenium.py

```diff
@@ -0,0 +1,49 @@
+import pytest 
+from selenium import webdriver
+import pytest
+import requests
+from dotenv import find_dotenv, load_dotenv
+from todoapp.app import MongoCall, create_app
+from selenium.webdriver.common.by import By
+from threading import Thread
+import os
+
+# Module scope re-uses the fixture 
+
+@pytest.fixture(scope='module')
+def client():
+    file_path = find_dotenv('.env')
+    load_dotenv(file_path, override=True)
+    os.environ["DATABASE"] = "e2edb"
+    os.environ["LOGIN_DISABLED"] = "true"
+    # Create the new app.
+    app = create_app()
+    thread = Thread(target=lambda: app.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield app
+        
+    # remove DB pymongo,mongoclient.
+
+@pytest.fixture(scope='module')
+def driver():
+    # path to your webdriver download 
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
If you skip specifying the path to the chromedriver, it will check both the PATH and the current folder. This will fix running the tests in Docker.
```suggestion
    with webdriver.Chrome(options=opts) as driver:
```

Reply:
Changed test name to e2e and included